crates/swc_html_minifier/src/lib.rs

```diff
@@ -11,6 +11,7 @@ use swc_common::{
     collections::AHashMap, comments::SingleThreadedComments, sync::Lrc, EqIgnoreSpan, FileName,
     FilePathMapping, Mark, SourceMap, DUMMY_SP,
 };
+use swc_ecma_transforms_base::perf::Items;
```

Comment:
Why is the imported?

Reply:
auto-insertion by IDE :smile: Not sure how it happens, Fixed